sparkplug-executor/src/test/scala/springnz/sparkplug/ExecutorServiceTests.scala

```diff
@@ -37,7 +63,12 @@ class ExecutorServiceTests(_system: ActorSystem)
 
     val clientActor = system.actorOf(Props(new Actor {
       override def receive = {
-        case ServerReady ⇒ probe forward ServerReady
+        case ServerReady ⇒
+          probe forward ServerReady
+          sender ! ClientReady
+
+        case Terminated(actor) ⇒
```

Comment:
Thanks for that observation. I meant for it to watch the server. 
I'll put that in as well.